lib/DDGC/DB/Result/InstantAnswer.pm

```diff
@@ -15,14 +15,14 @@ sub u { [ 'InstantAnswer', 'view', $_[0]->id ] }
 
 column id => {
 	data_type => 'text',
-    for_endpt => 1
 };
 primary_key 'id';
 
 # editable ID
 column meta_id => {
     data_type => 'text',
-    for_endpt => 1
+    for_endpt => 1,
+    alias => 'id'
```

Comment:
yeah, new custom attribute :) it's used to display meta_id as id in the JSON endpoint which use the TO_JSON method to get the columns, such as ia_json